Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -156,7 +156,7 @@ bool AVIDump::CreateVideoFile()
   s_codec_context->time_base.num = 1;
   s_codec_context->time_base.den = VideoInterface::GetTargetRefreshRate();
   s_codec_context->gop_size = 12;
-  s_codec_context->pix_fmt = g_Config.bUseFFV1 ? AV_PIX_FMT_BGRA : AV_PIX_FMT_YUV420P;
+  s_codec_context->pix_fmt = g_Config.bUseBGRA ? AV_PIX_FMT_BGRA : AV_PIX_FMT_YUV420P;
```

Comment:
Why do we need to specify this ourself at all? We controll the input format, everything else should be handled by ffmpeg.